src/main/webapp/app/shared/api/manual/evidence-api.ts

```diff
@@ -1,16 +1,17 @@
-import { getEvidence } from 'app/shared/util/core-evidence-submission/core-evidence-submission';
```

Comment:
I forgot to ask in the previous PRs. Why don't we automatically generate these clients?

Reply:
I believe it was because the legacy API isn't on the swagger pages.